docs/app/Examples/modules/Checkbox/Types/CheckboxExampleShorthand.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { Checkbox, Divider } from 'semantic-ui-react'
+
+const CheckboxExampleCheckbox = () => (
+  <div>
+    <Checkbox label={{ children: 'Prop label', className: 'foo' }} />
+    <Divider />
+    <Checkbox label={<label><b>Element label</b></label>} />
```

Comment:
We're listing each shorthand as separate examples.  Ideally, shorthand examples immediately follow the example showing the regular usage.  I think we should split this into two examples and show them immediately after the first Checkbox example:

![image](https://cloud.githubusercontent.com/assets/5067638/20803609/9ad0678e-b7a4-11e6-9430-2ef884b2d956.png)